### PR TITLE
SCRIPT-Style: Introduce New Color Palette for UI

### PR DESCRIPTION
**Pull request by samwise-repo1**
SCRIPT-Localization support has been added, with initial translations for Spanish, French, and German.

### CODE REVIEW

File: samwise.js [hunk 1]
```diff
@@ -1,46 +1,46 @@
 /**
```
Comment (samwise-repo1) [inline samwise.js:1]: SCRIPT-Consider renaming this variable to something more descriptive for better readability.

Comment (frodo-repo) [inline samwise.js:1]: SCRIPT-Well done on adding type checking, it really enhances the code safety.

Comment (gandalf-repo) [inline samwise.js:1]: SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.

Comment (samwise-repo1) [inline samwise.js:1]: SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.

Comment (samwise-repo1) [inline samwise.js:1]: SCRIPT-Consider renaming this variable to something more descriptive for better readability.

**Pull request merged**